content/reference/dos-mitigation.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "DOS Mitigation"
+weight: 4
+---
+
+In general we have to design our protocols to be resilient to malicious or
+misbehaving nodes. libp2p offers reasonable defaults and primitives tools let the
+end application developer design resilient peer-to-peer programs.
+
+# General tactics that work across libp2p implementations
```

Comment:
Looks like we need to add a level of indentation throughout given the rendering in https://bafybeiegvuphbhruoy6ntjhgh5xfmweye7ubeshinyv5lugdhgxml6dice.on.fleek.co/reference/dos-mitigation/